packages/honeycomb-opentelemetry-web/examples/hello-world-custom-exporter/index.js

```diff
@@ -0,0 +1,77 @@
+import {
+  getDefaultTraceExporter,
+  HoneycombWebSDK,
+} from '@honeycombio/opentelemetry-web';
+import { trace } from '@opentelemetry/api';
+import { getWebAutoInstrumentations } from '@opentelemetry/auto-instrumentations-web';
+import { ZoneContextManager } from '@opentelemetry/context-zone';
+import { ConsoleSpanExporter } from '@opentelemetry/sdk-trace-base';
+
+const configDefaults = {
+  ignoreNetworkEvents: true,
+};
+
+const main = () => {
+  const hcoConfig = {
+    // defaults to sending to US instance of Honeycomb
+    // endpoint: "https://api.eu1.honeycomb.io/v1/traces", // uncomment to send to EU instance
+    apiKey: 'api-key-goes-here',
+  };
+
+  // Initialize Honeycomb SDK
+  const sdk = new HoneycombWebSDK({
+    ...hcoConfig,
+    serviceName: 'hny-web-distro-example:hello-world-web', // Replace with your application name. Honeycomb will name your dataset using this variable.
+    debug: true,
+    instrumentations: [
+      getWebAutoInstrumentations({
+        // load custom configuration for xml-http-request instrumentation
+        '@opentelemetry/instrumentation-xml-http-request': configDefaults,
+        '@opentelemetry/instrumentation-fetch': configDefaults,
+        '@opentelemetry/instrumentation-document-load': configDefaults,
+      }),
+    ],
+    contextManager: new ZoneContextManager(),
+    webVitalsInstrumentationConfig: {
+      vitalsToTrack: ['CLS', 'FCP', 'FID', 'INP', 'LCP', 'TTFB'],
+    },
+    traceExporters: [
+      // comment out to only use the console exporter
+      getDefaultTraceExporter(hcoConfig),
+      new ConsoleSpanExporter(),
```

Comment:
One thing I toyed with a bit was just always including the default exporter. It seems some people want the ability to override it so I added this as a way to. But I could also see another config option, such as `excludeDefaultExporter` that defaults to false.

Reply:
We discussed this and we are simply moving forward with the list of trace exporters and will add overriding the default exporter in a new PR.